tests/cypress/integration/case_7_image_scale_roi.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check if image was scaled to ROI', () => {
+
+    const caseId = '7'
+    const labelName = `Case ${caseId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`
+    const image = `${imageFileName}.png`
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.imageGenerator('cypress/fixtures', image, width, height, color, posX, posY, labelName)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, image)
+        cy.openTaskJob(taskName)
+    })
+
+    describe(`Testing "${labelName}"`, () => {
+        it('Create ROI', () => {
+            cy.get('.cvat-resize-control')
+            .click()
+            cy.get('.cvat-canvas-container')
+            .trigger('mousedown', 309, 431, {which: 1})
+            .trigger('mousemove', 616, 671)
+            .trigger('mouseup', 616, 671)
+        })
+        it('Image scaled to ROI', () => {
+            cy.get('#cvat_canvas_background')
+            .should('have.attr', 'style').and('contain', 'scale(3.2817') //Chrome: scale(3.28173), Firefox: scale(3.28172)
```

Comment:
I suppose this value depends on browser implementation and can be changed itself. Let's just check that scale is equal to 1 before and not equal 1 after.

Reply:
Done. Made a check of the original scale and check that after scaling the scale does not contain the original.

Reply:
Hmm, again, hardcoding this value it is not the best way I believe.
Can we get the value 1.065 in runtime and than check if it was increased after zoom?

Reply:
Changed it to get the initial value and compare it with the final one. Thanks.